CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 ### Feature
 
+- Added link integrity potential breakage warning message when deleting a referenced page @danielamormocea
+- Added new components & interfaces for content-rules `Rules` control in Volto. Rules management in both controlpanel and object view. @andreiggr
+- Added portrait middleware adapter. @instification
```

Comment:
@instification Careful, it looks like you picked up some unrelated changelog entries while merging/rebasing

Reply:
I fixed it.